src/imars3d/backend/autoredux.py

```diff
@@ -62,11 +62,18 @@ def extract_info_from_path(data_file: str) -> dict:
     ----------
     data_file
         The data file to extract information from.
+        ex. /HFIR/CG1D/IPTS-25777/raw/ct_scans/iron_man/20191029_ironman_small_0070_000_000_0002.tiff
 
     Returns
     -------
     dict
         The extracted information.
     """
+    extracted_data = data_file.split("/")
+    data_dict = {}
+    # index from right to be agnostic of root
+    data_dict["facility"] = extracted_data[-6]  # "HFIR"
```

Comment:
I have mixed feelings on the magic numbers. Can we add some unit tests that hit this function directly?

Reply:
https://github.com/ornlneutronimaging/iMars3D/pull/243/files#diff-7f2d180c538e163ec2574cc906a984fa641ac1c75dc14b260ea31d47ab7312d5R22

could put these in vars but whats the likelihood for them to change or be used elsewhere? Its commented with and example and expected result so there's intuition as for why.